### ISSUE TITLE
Cannot use Redux that is embedded into a theme (not loaded as plugin)

### ISSUE THREAD

**Issue by LAccorinti** [reduxframework/redux-framework#2736]
I have a plugin that use TGMPA to load Redux if not present.
I have a theme that has Redux embedded INSIDE the theme.
Is it possible to use the Redux embedded into the them without the need to load the Redux from plugin?
To avoid Redux to be loaded two times (one from the theme and one from the plugin).
I looked into docs but found nothing about this...
Thanks

**Comment by kprovance** [comment]
I'm not sure I understand the question.  You should really use one or the other.  Both is a waste of space.  Be aware, the plugin will ALWAYS override the embed.  If you are using the plugin, you don't need to embed.  And yes, you can embed, in which case you do not need the plugin.

**Comment by LAccorinti** [comment]
The fact is that the plugin is developed by myself (so I decided to use Redux loaded as plugin), but the theme is not mine, is downloaded from Wordpress theme repository.
This theme contain Redux Framework "into himself", loaded from his assets folders.
The question is: can I use the Redux framework instance that is loaded from the theme to display my plugin admin panels or I need to use the Redux Plugin loaded from WP repository?
And next: there is a documentation to explain how to do it? I checked "if the Redux class exists" but doesn't work...
thanks
